### DISCUSSION TITLE
Route resolution / page loading with fetch: lazy

### DISCUSSION THREAD

**Discussion by TroyanOlga** [nuxt/nuxt#23290] [Questions]
When I click on nuxtLink, I immediately see the previous content disappearing while my page is loading and don't see any new content until all data is loaded. I use useFetch with "lazy" option to prevent route resolution blocking. I would expect in this case to immediately see new page's content with the placeholders until requests are resolved. Is my expectation wrong here? Anything else I can check to ensure the page loads immediately and doesn't wait for the requests?

**Comment by xjccc** [comment]
[docs of useLazyFetch](https://nuxt.com/docs/api/composables/use-lazy-fetch#example)
